VsixPackage/source.extension.vsixmanifest

```diff
@@ -1,24 +1,24 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
-  <Metadata>
-    <Identity Id="VSOutputEnhancer.Nikolay Balakin.a06be4c3-f97e-425c-8a0d-bdef08ac2abb" Version="1.6" Language="en-US" Publisher="Nikolay Balakin" />
-    <DisplayName>Output enhancer</DisplayName>
-    <Description xml:space="preserve">Extension to add colors to Visual Studio output window.</Description>
-    <License>License.txt</License>
-    <Icon>Icon.png</Icon>
-    <PreviewImage>Preview.png</PreviewImage>
-  </Metadata>
-  <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[11.0,16.0)" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[11.0,16.0)" />
-  </Installation>
-  <Dependencies>
-    <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
-  </Dependencies>
-  <Assets>
-    <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="VSOutputEnhancer" Path="|VSOutputEnhancer|" />
-  </Assets>
-  <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0.25914.00,16.0)" DisplayName="Visual Studio core editor" />
-  </Prerequisites>
+    <Metadata>
+        <Identity Id="VSOutputEnhancer.Nikolay Balakin.a06be4c3-f97e-425c-8a0d-bdef08ac2abb" Version="1.7" Language="en-US" Publisher="Nikolay Balakin" />
+        <DisplayName>Output enhancer</DisplayName>
```

Comment:
Sorry for the whitespace changes. I edited in the VSIX editor and it added them.